### PR TITLE
Add Vue integration docs

### PR DESCRIPTION
**Pull request by jacekbogdanski**
**Important**

Before merging this PR we need to update `package.json` file with published `ckeditor4-vue` npm package. Currently, it uses a dev repository with the post-install script for development purposes. So, instead of: 
https://github.com/ckeditor/ckeditor4-docs/blob/76aeec54bd26d55400298d5c5fbf3cf369d6b90c/package.json#L26
https://github.com/ckeditor/ckeditor4-docs/blob/76aeec54bd26d55400298d5c5fbf3cf369d6b90c/package.json#L68

we should have:
```
"ckeditor4-vue": "0.0.1"
```

Closes ckeditor/ckeditor4-vue#7.
Closes ckeditor/ckeditor4#3774.

### CODE REVIEW

File: docs/guide/dev/integration/vue/README.md [hunk 1]
```diff
@@ -12,6 +12,335 @@ For licensing, see LICENSE.md.
...
+```plaintext
+npm install ckeditor4-vue
+```
+
+Now you need to enable the CKEditor Vue component in your application. There are 2 ways to do so:
+
+* [via a direct script include](#direct-script-include),
+* [by using ES6 module imports](#using-es6-modules).
+
+Optionally, you can [use the component locally](#using-component-locally).
```
Comment (f1ames) [inline docs/guide/dev/integration/vue/README.md]: Using locally is not an alternative to direct script include or module imports but to `Vue.use` so I'm not sure if it should be listed here. Mentioning it in both sections as already done (with `info-box`) seems to be enough IMHO.

File: docs/guide/dev/integration/vue/README.md [hunk 2]
```diff
@@ -12,6 +12,335 @@ For licensing, see LICENSE.md.
...
+		}
+	}
+</script>
+```
+
+## Component directives
+
+### `editorUrl`
+
+By default, the CKEditor 4 Vue component loads the [standard preset](https://ckeditor.com/cke4/presets-all) of the latest CKEditor 4 release from the [CDN](https://cdn.ckeditor.com/) when creating the first editor. This behavior can be altered by changing the value of the `editorUrl` directive to point to the desired CKEditor script location:
```
Comment (f1ames) [inline docs/guide/dev/integration/vue/README.md]:
```suggestion
By default, the CKEditor 4 Vue component loads the [standard-all preset](https://ckeditor.com/cke4/presets-all) of the latest CKEditor 4 release from the [CDN](https://cdn.ckeditor.com/) when creating the first editor. This behavior can be altered by changing the value of the `editorUrl` directive to point to the desired CKEditor script location:
```

**Comment by jacekbogdanski** [comment]
> I had some npm i issue due to package-lock.json file:
I can't reproduce it. Maybe it's not related to PR changes?

> Also do we need:
> docs/sdk/examples/assets/vue/main.js
> docs/sdk/examples/assets/vue/main.js.map

We don't. I forgot to update .gitignore for these files.

> And when switching to NPM package we will get rid of scripts/integrations/vue/webpack.config.js right?

Not really. It's building sample code, not integration. You can see the same approach with React and Angular sample apps.

> Something strange with the sample menu styling going on (Chrome 79 on Linux Mint 19.2, FF seems to be working fine) thinking

Seems like Vue treats HTML standards very seriously and doesn't trim empty spaces between list tags like it's done in HTML specification: https://github.com/vuejs/vue/issues/7701

> Can we also polish the styling of this example:

Honestly, it looks fine for me, but I don't mind updating it. However, maybe it makes more sense to extract it as a separate issue, so we can unify examples between Vue, React and Angular? Otherwise, we will end up with 3 different samples sharing a lot of similarities instead of a standardized approach.

**Comment by f1ames** [comment]
Rebased onto latest \`stable\`.

**Comment by f1ames** [comment]
## **WARNING**: Before merging this PR we need to update package.json file with published ckeditor4-vue npm package (see https://github.com/ckeditor/ckeditor4-docs/pull/283#issue-361341805).

☝️ Means it could be merged only after NPM package is published and should be done after Vue package release (by @f1ames or @jacekbogdanski).

**Comment by f1ames** [comment]
Rebased onto latest `stable`.

**Pull request merged**